tests/unit/all.js

```diff
@@ -23,7 +23,9 @@ define([
 	"./ToasterMessage-insert-show-hide-remove",
 	"./Toggle",
 	"./Button",
+	"./Button-bidi",
```

Comment:
for easier access to all bidi tests (and consistency with the fact bidi code is in bidi dir) maybe it should be bidi/Button and bidi/ToggleButton here.